packages/nimble/R/makevars.R

```diff
@@ -64,7 +64,14 @@ function(target, vars = character(), .useLib = UseLibraryMakevars)
                                    else
                                      "Makevars", package = "nimble")
 
+    cppad_inc <- nimbleOptions("CppAD_directory")
+    # if(is.na(cppad_inc)) cppad_inc <- ""
+    # else cppad_inc <- paste0("-I\"",cppad_inc,"\"")
+    # browser()
+    # cppad_inc <- paste0("-I ", cppad_inc)
+    cppad_inc <- '-I"/Users/iateb/Documents/cppad/cppad-20170413"'
```

Comment:
Since we only expect this to matter if `nimbleOptions('enableDerivs')==TRUE`, we could make this `cppad_inc` <- paste0("'-I\\"", nimbleOptions('CppADdir'), "\\"'")`